doc/source/code_ref/coordinates.rst

```diff
@@ -0,0 +1,196 @@
+SunPy coordinates
+=================
+
+The SunPy coordinates submodule is an implementation of the common solar physics
+coordinate frames in to the :ref:`Astropy coordinates system <astropy:astropy-coordinates>`.
+
+.. warning::
+
+   The accuracy of the transformations in this module have not been rigorously
+   verified. They have been compared to `sunpy.wcs` and match to numerical
+   precision. Independent verification will be added at a later date.
+
+
+Getting Started
+---------------
+
+The easiest interface to the coordinates module is through the `~astropy.coordinates.SkyCoord` class::
+
+  >>> import astropy.units as u
+  >>> from astropy.coordinates import SkyCoord
+  >>> import sunpy.coordinates
+  >>> c = SkyCoord(-100*u.arcsec, 500*u.arcsec, frame='helioprojective')
+  >>> c = SkyCoord(90*u.deg, -76*u.deg, frame='helioprojectiveradial')
+  >>> c = SkyCoord(x=-72241.0*u.km, y=361206.1*u.km, z=589951.4*u.km, frame='heliocentric')
+  >>> c = SkyCoord(70*u.deg, -30*u.deg, frame='heliographicstonyhurst')
+  >>> c
```

Comment:
Wondering whether an underscore would improve readability in ``heliographicstonyhurst`` -> ``heliographic_stonyhurst``